src/transformers/models/plbart/tokenization_plbart.py

```diff
@@ -374,6 +387,8 @@ def _build_translation_inputs(
         """Used by translation pipeline, to prepare inputs for the generate function"""
         if src_lang is None or tgt_lang is None:
             raise ValueError("Translation requires a `src_lang` and a `tgt_lang` for this model")
+        src_lang = self._convert_lang_code_special_format(src_lang)
+        tgt_lang = self._convert_lang_code_special_format(tgt_lang)
         self.src_lang = src_lang
```

Comment:
```suggestion
        self.src_lang = self._convert_lang_code_special_format(src_lang)
        tgt_lang = self._convert_lang_code_special_format(tgt_lang)
```
Quick question, why don't we update both `tgt` and `src`